wallet/lib/model/nft.dart

```diff
@@ -82,9 +82,7 @@ class NFT extends Equatable {
       final cookbook = await walletsStore.getCookbookById(cookbookID);
       ownerAddress = cookbook?.creator ?? "";
 
-      if (owner.isEmpty && type != NftType.TYPE_RECIPE) {
-        owner = await walletsStore.getAccountNameByAddress(ownerAddress);
-      }
+      owner = await walletsStore.getAccountNameByAddress(ownerAddress);
```

Comment:
Whats the reason for removing?

Reply:
this condition restricting owner variable to get initialised, so the user is unable to view owner name in ownership tab in purchase nft screen. 


<img width="460" alt="image" src="https://user-images.githubusercontent.com/115459890/204331897-fcf29e24-dc01-454d-b9a0-d15a70842b37.png">